### ISSUE TITLE
TypeError: context is undefined

### ISSUE THREAD

**Issue by fayf** [transformsai/UnityGrpcWeb#3]
I'm currently trying to use this like so:
```
var channel = UnityGrpcWeb.MakeChannel(
    "http://localhost:9090",
    GrpcWebMode.GrpcWeb,
    null,
    true,
    true
);
client = new Greeter.GreeterClient(channel);
```

However, I get the following error after trying to run the WebGL build in the browser:
```
TypeError: context is undefined
    _GetGlobalObject http://localhost:61817/Build/grpc-test-client.framework.js:1937
    createExportWrapper
    ...
```

Some cursory digging later, I traced it to the JS code in this snippet:
```
GetGlobalObject: function (returnTypeId, name){
try{
  name = Pointer_stringify(name);
  var context = Module['UnityJsInteropInstance'];
  var ret = context.GetGlobalObject(name);;
  setValue(returnTypeId, ret.type, 'i32');
  return ret.value;
}
...
```
and `context` is resolving to `undefined`. I don't really know how to troubleshoot further. Am I doing something wrong?

**Comment by fayf** [comment]
Forgot to mention that these are the versions I'm using:
- Unity: `2021.3.9f1` (includes the fix for #2)
- `Grpc.Net` packages: Tried the version combination specified as dependencies in this package (`2.42.0` / `2.41.0`), as well as `2.48.0`.
- Firefox: `104.0.1`
- Chrome: `105.0.5195.52`

**Comment by fayf** [comment]
I managed to fix the problem. It was a code stripping issue with `RuntimeRaw.Initialize` getting stripped. However, I am still unable to get things working.
